navig/MsgOdometry odometry

navig/MsgObjectPosition pos

navig/MsgPoint2d center

float64 width_ratio

int32 frame_number